migration-guide.md

```diff
@@ -0,0 +1,134 @@
+# blockstack.js to Stacks.js migration guide
+
+This guide will help migrate your Blockstack app from blockstack.js to the new Stacks.js packages and Connect.
+
+## Auth
+
+The main change for auth is that the Stacks Connect library has replaced the `redirectToSignIn` function from blockstack.js. 
+Instead of redirecting to the now deprecated Blockstack Browser, the authentication flow is completed within a popup window using
+the new authenticator app.
+You can still use the API in `@stacks/auth` to create custom auth requests manually if desired.
+
+### Using blockstack.js
+```typescript
+import { UserSession, AppConfig } from 'blockstack';
+
+// Configuring your app
+const appConfig = new AppConfig()
+const userSession = new UserSession({ appConfig })
+
+// Initiating auth flow
+if(!userSession.isUserSignedIn()) {
+  userSession.redirectToSignIn();
+}
+
+// Handling sign in
+if (userSession.isSignInPending()) {
+  userSession.handlePendingSignIn().then((userData) => {
+    window.history.replaceState({}, document.title, "/");
+    this.setState({ userData: userData});
+  });
+}
+```
+
+### Using Blockstack Connect
+See full tutorial [here](https://docs.blockstack.org/authentication/connect)
+```typescript
+
+// Configuring your app
+import { Connect } from '@stacks/connect';
+
+const authOptions = {
+  redirectTo: '/',
+  finished: ({ userSession }) => {
+    console.log(userSession.loadUserData());
+  },
+  appDetails: {
+    name: 'My Cool App',
+    icon: 'https://example.com/icon.png',
+  },
+};
+
+const App = () => <Connect authOptions={authOptions}>// the rest of your app's components</Connect>;
```

Comment:
Is there a clean, non JSX alternative here?

Reply:
Yes, the non-JSX alternative is using `showBlockstackConnect(authOptions)` as described below. I'll clarify.